packages/kit/src/core/make_package/index.js

```diff
@@ -63,9 +63,8 @@ export async function make_package(config, cwd = process.cwd()) {
 				? (await preprocess(source, config.preprocess, { filename })).code
 				: source;
 		} else if (ext === '.ts' && !file.endsWith('.d.ts')) {
-			// TODO transpile TS file and emit types
-			// also, we want to emit types from JSDoc annotations in .js files
-			throw new Error('svelte-kit package does not yet support TypeScript');
+			out_file = file.slice(0, -'.ts'.length) + '.js';
```

Comment:
Can't we just do this?
```suggestion
			out_file = `${file.slice(0, -3)}.js`;
```

Reply:
or
```suggestion
			out_file = file.replace(/ts$/, 'js');
```

Reply:
We could, but for me ` -'.ts'.length` is the most readable (I know what is substracted); and it's consistent with the approach taken a few lines above.